themes/bootstrap3/templates/holds/edit.phtml

```diff
@@ -0,0 +1,89 @@
+<?php
+  // Set page title.
+  $this->headTitle($this->translate('hold_edit_title'));
+
+  // Set up breadcrumbs:
+  $this->layout()->breadcrumbs = '<li><a href="' . $this->url('myresearch-home') . '">' . $this->transEsc('Your Account') . '</a></li>'
+    . '<li><a href="' . $this->url('holds-list') . '">' . $this->transEsc('My Holds') . '</a></li>'
+    . '<li class="active">' . $this->transEsc('hold_edit_title') . '</li>';
+?>
+<h2><?=$this->transEsc('hold_edit_title')?></h2>
+
+<form action="<?=$this->url('holds-edit')?>" class="form-edit-holds" method="post" name="editHolds">
+  <input type="hidden" name="csrf" value="<?=$this->escapeHtmlAttr($this->auth()->getManager()->getCsrfHash())?>">
+  <?=$this->flashmessages()?>
+
+  <?php foreach ($this->selectedIDS as $id): ?>
+    <input type="hidden" name="selectedIDS[]" value="<?=$this->escapeHtmlAttr($id)?>">
+  <?php endforeach; ?>
+
+  <?php if (in_array('frozen', $this->fields)): ?>
```

Comment:
I confess that I haven't tried to work this out from either looking at the code or interacting with the interface, but I'm curious what the relationship is between frozen and frozenUntil. Are there any interdependencies between them? I can imagine scenarios where an ILS supports one or the other or both... but this may lead to some edge cases where one takes precedence over the other, or some extra rules might come into play. Do we need to worry about any of that?

Reply:
This needs to be documented, but as I see it frozenUntil can be supported alongside frozen, but not without it.